### PR TITLE
ENT-4238: Better logging for collect_window

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @vpodzime can review this?

File: cf-serverd/server_transform.c [hunk 1]
```diff
@@ -376,7 +376,7 @@ static void KeepControlPromises(EvalContext *ctx, const Policy *policy, GenericA
                 COLLECT_WINDOW = (int) IntFromString(value);
                 Log(LOG_LEVEL_VERBOSE,
                     "Setting collect_window to %d (seconds)",
-                    COLLECT_INTERVAL);
+                    COLLECT_WINDOW);
```
Comment (vpodzime) [inline cf-serverd/server_transform.c]: window**s** again, this time in the commit message.

File: cf-serverd/cf-serverd-functions.c [hunk 2]
```diff
@@ -619,7 +619,8 @@ static void CollectCallIfDue(EvalContext *ctx)
         {
             ConnectionInfo *info = ConnectionInfoNew();
             assert(info);
-
+            Log(LOG_LEVEL_DEBUG,
+                "Server has %d seconds to complete report collection ()collect_window)", COLLECT_WINDOW);
```
Comment (vpodzime) [inline cf-serverd/cf-serverd-functions.c]: Extra closing bracket.

Comment (vpodzime) [reply cf-serverd/cf-serverd-functions.c]: Maybe say *Hub* instead of *Server*?

File: cf-serverd/server_transform.c (hunk 1 above)
Comment (nickanderson) [reply cf-serverd/server_transform.c]: BLARG, thanks

File: cf-serverd/cf-serverd-functions.c (hunk 2 above)
Comment (nickanderson) [reply cf-serverd/cf-serverd-functions.c]: Good idea.

**Comment by nickanderson** [comment]
@vpodzime as you wish

**Pull request merged**